ethsigner/signer/raw/src/main/java/tech/pegasys/ethsigner/signer/raw/RawSubCommand.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.raw;
+
+import static tech.pegasys.ethsigner.DefaultCommandValues.MANDATORY_HOST_FORMAT_HELP;
+
+import tech.pegasys.ethsigner.SignerSubCommand;
+import tech.pegasys.ethsigner.TransactionSignerInitializationException;
+import tech.pegasys.ethsigner.core.signing.SingleTransactionSignerProvider;
+import tech.pegasys.ethsigner.core.signing.TransactionSigner;
+import tech.pegasys.ethsigner.core.signing.TransactionSignerProvider;
+import tech.pegasys.ethsigner.signer.filebased.CredentialTransactionSigner;
+
+import com.google.common.base.MoreObjects;
+import org.web3j.crypto.Credentials;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+/** Raw sub-command - DO NOT USE IN PRODUCTION, Private Key is in plain text */
+@Command(
+    name = RawSubCommand.COMMAND_NAME,
+    description =
+        "Sign transactions with an unecrypted key specified on cmdline. NOT SUITABLE FOR PRODUCTION.",
```

Comment:
Don't think we want add something that we don't actually want people use. Could be hidden I suppose?

Reply:
hidden is possible - so have forced that in